src/components/Profile.js

```diff
@@ -0,0 +1,37 @@
+import React from 'react';
+import { useSelector } from 'react-redux';
+import { ListGroup } from 'react-bootstrap';
+
+const Profile = () => {
+  const rockets = useSelector((state) => state.rocketsRducer);
+  const registeredRocket = rockets.filter((rocket) => rocket.reserved);
+  const missions = useSelector((state) => state.MissionReducer);
+  const registerMisson = missions.filter((mission) => mission.reserved);
+
+  return (
+    <div>
+      <div className="d-flex justify-content-around">
+        <div>
+          <h2>My Rockets</h2>
+          <ListGroup>
+            {registeredRocket.map((rocket) => (
+              <ListGroup.Item key={rocket.id}>{rocket.title}</ListGroup.Item>
+            ))}
+          </ListGroup>
+        </div>
+        <div>
+          <h2>My Missiosn </h2>
```

Comment:
Taking care of the spelling is also important in the graphic interface. A project with misspellings might appear unprofessional to recruiters.